.travis.yml

```diff
@@ -2,16 +2,18 @@ language: elixir
 elixir:
   - 1.3.1
   - 1.4.0
+  - 1.6.6
+  - 1.7.3
 otp_release:
   - 17.5
   - 18.0
   - 19.0
 before_script:
   - MIX_ENV=test mix do deps.get
 script:
-  - MIX_ENV=test mix test --no-start
```

Comment:
why was this needed?

Reply:
the `--no-start` prevents from starting any extra applications that would be defined... now that with newer elixir versions it is commonly used you can't keep this flag anymore